src/Image/__stories__/Image.stories.js

```diff
@@ -15,10 +15,4 @@ storiesOf('Components|Image', module)
         <Image disableStyles
             photo='https://content.fortune.com/wp-content/uploads/2019/07/hippocorn.jpg'
             size='l' />
-    ))
-    .add('custom styles', () => (
-        <Image
-            customStyles={require('../../utils/WithStyles/customStylesTest.css')}
-            photo='https://content.fortune.com/wp-content/uploads/2019/07/hippocorn.jpg'
```

Comment:
just so we know, this is what we're losing :(
![hippocorn](https://content.fortune.com/wp-content/uploads/2019/07/hippocorn.jpg)